src/lib.rs

```diff
@@ -21,6 +21,7 @@ mod utils;
 pub use error::{ParseError, RequestError, Fault};
 pub use request::{Request, RequestResult};
 pub use value::Value;
+pub use parser::parse_response;
```

Comment:
`parse_response` returns a `ParseResult`, which isn't public, so that would need to be made public, too.